ewlegal-utils.sty

```diff
@@ -0,0 +1,35 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+\ProvidesPackage{engwales-utils}[2021-02-21 England and Wales Legal Utilities]
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Variables
+%%%%%
+
+%% Sets optional options of the point env
+% \newcommand{\pointspacing}{leftmargin=0cm,labelsep=1cm}
+% TODO This work around to reusing the point env does not work. FIX!
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typeset Elements
+%%%%%
+
+%% Implement tramlines for use in titles, element is not centred, or bolded
```

Comment:
what does

> element is not centred, or bolded

refer to?

Reply:
The tramlined element. It is merely a reminder that if you are using it, you have to centre, or bold it yourself.

Reply:
I was trying to reduce the limitations of using the element in ways we aren't dictating. As it gets wrapped in centers for the titles anyways, I thought it doesn't need it.

Reply:
yeah i think leaving that to the user is good. closing.